<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="selectMethodLabel">
       <property name="text">
        <string>Wybierz sposób wskazania obszaru:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="selectMethodCb"/>
     </item>
     <item>
      <widget class="QLabel" name="selectLayerLabel">
       <property name="text">
        <string>Warstwa:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsMapLayerComboBox" name="selectLayerCb">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="selectLayerFeatsCb">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Tylko zaznaczone obiekty [0]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectAreaBtn">
       <property name="text">
        <string>Wskaż obszar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
